docs/book/stacks-and-components/stack-deployment/cloud-stacks/minimal-gcp-stack.md

```diff
@@ -2,29 +2,37 @@
 description: A simple guide to quickly set up a minimal stack on GCP.
 ---
 
-# Set up minimal GCP stack
-
-Here's a 7-step guide to get a production-ready GCP stack.
+# Set up a minimal GCP stack
 
+This page aims to quickly set up a minimal production stack on gcp. With just a 
+few simple steps you will set up a service account with specifically scoped 
+permissions that ZenML can use to authenticate with the relevant GCP resources.
 
 
 ### 1) Choose a GCP project&#x20;
 
-In the Google Cloud console, on the project selector page, select or [create a Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+In the Google Cloud console, on the project selector page, select or 
+[create a Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+Make sure a billing account is attached to this project to allow the use of 
+some APIs.
+
+```bash
```

Comment:
do you need to include some wording to the effect of "alternatively, if you want to create this using the CLI, use the following command" etc?